spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -1197,6 +1197,30 @@ describe "macro methods" do
     end
   end
 
+  describe "proc notation tests" do
```

Comment:
I think that `tests` is redundant here

Reply:
Changed to "methods" to fit in with other tests.